Error on Display Owner

Caused by: org.attoparser.ParseException: Exception evaluating SpringEL expression: "pet.type" (template: "owners/ownerDetails" - line 52, col 21)
	at org.attoparser.MarkupParser.parseDocument(MarkupParser.java:393) ~[attoparser-2.0.5.RELEASE.jar:2.0.5.RELEASE]
	at org.attoparser.MarkupParser.parse(MarkupParser.java:257) ~[attoparser-2.0.5.RELEASE.jar:2.0.5.RELEASE]
	at org.thymeleaf.templateparser.markup.AbstractMarkupTemplateParser.parse(AbstractMarkupTemplateParser.java:230) ~[thymeleaf-3.0.11.RELEASE.jar:3.0.11.RELEASE]
	... 55 common frames omitted
Caused by: org.thymeleaf.exceptions.TemplateProcessingException: Exception evaluating SpringEL expression: "pet.type" (template: "owners/ownerDetails" - line 52, col 21)

--- a/pet-clinic-data/src/main/java/com/github/mathan26/petclinic/model/PetType.java
+++ b/pet-clinic-data/src/main/java/com/github/mathan26/petclinic/model/PetType.java
@@ -15,4 +15,9 @@ import lombok.*;
 public class PetType extends BaseEntity {
     @Column(name="name")
     private String name;
+
+    @Override
+    public String toString() {
+        return  name;
+    }
 }

--- a/pet-clinic-web/src/main/java/com/github/mathan26/petclinic/bootstrap/DataLoader.java
+++ b/pet-clinic-web/src/main/java/com/github/mathan26/petclinic/bootstrap/DataLoader.java
@@ -60,7 +60,7 @@ public class DataLoader implements CommandLineRunner {
         owner1.setAddress("123 Brickerel");
         owner1.setCity("Miami");
         owner1.setTelephone("1231231234");
-        ownerService.save(owner1);
+
 
         Pet mikesPet = new Pet();
         mikesPet.setPetType(savedDogPetType);
@@ -69,6 +69,7 @@ public class DataLoader implements CommandLineRunner {
         mikesPet.setName("Rosco");
         owner1.getPets().add(mikesPet);
 
+        ownerService.save(owner1);
 
         Owner owner2 = new Owner();
         owner2.setFirstName("Fiona");

--- a/pet-clinic-web/src/main/resources/application.properties
+++ b/pet-clinic-web/src/main/resources/application.properties
@@ -5,3 +5,4 @@ spring.profiles.active=springdatajpa
 
 # Internationalization
 spring.messages.basename=messages/messages 
+spring.datasource.url=jdbc:h2:mem:testdb

--- a/pet-clinic-web/src/main/resources/templates/owners/ownerDetails.html
+++ b/pet-clinic-web/src/main/resources/templates/owners/ownerDetails.html
@@ -49,7 +49,7 @@
                 <dd
                         th:text="${#temporals.format(pet.birthDate, 'yyyy-MM-dd')}" /></dd>
                 <dt>Type</dt>
-                <dd th:text="${pet.type}" /></dd>
+                <dd th:text="${pet.petType}" /></dd>
             </dl>
         </td>
         <td valign="top">

